### ISSUE TITLE
Unavailable Font Error

### ISSUE THREAD

**Issue by dharm187** [Microsoft/azure-maven-plugins#2051]
### Plugin name and version
azure-webapp-maven-plugin 1.6.0

### Plugin configuration in your `pom.xml`
<configuration>
          <schemaVersion>V2</schemaVersion>  
          <resourceGroup>XXXX</resourceGroup>  
          <appName>XXXXX</appName>  
          <region>australiasoutheast</region>  
          <pricingTier>P1V2</pricingTier>  
            
          <appSettings> 
            <property> 
              <name>JAVA_OPTS</name>  
              <value>-Dserver.port=80</value> 
            </property> 
          </appSettings>  
            
          <runtime> 
            <os>linux</os>  
            <javaVersion>jre8</javaVersion>  
            <webContainer>TOMCAT 8.5</webContainer> 
          </runtime>  
          <deployment> 
            <resources> 
              <resource> 
                <directory>${project.basedir}/target</directory>
                <targetPath>abc/xyz</targetPath>
                <includes> 
                  <include>*.war</include> 
                </includes> 
              </resource> 
            </resources> 
          </deployment>
        </configuration> 

### Expected behavior
Excel file should be generated

### Actual behavior

While using a latest tomcat version, we are getting error while generating excel. Below is the error:

Probable fatal error:No fonts found

When working with older version of tomcat, for ex. tomcat 8.5.57, the excel generation is working good. 

### Steps to reproduce the problem

Try to generate excel using Apache POI library and latest version of Tomcat. Error will be generated

![image](https://user-images.githubusercontent.com/25227107/172825579-d08dde0f-1c65-43c6-99da-f0f17218c791.png)

**Comment by Flanker32** [comment]
@dharm187 Thanks for your report. Could you plese share us a demo project to reproduce this issue? By the way, could this issue be reproduced locally with the same tomcat container in Azure?

**Comment by dharm187** [comment]
@Flanker32 Yes. I think this can be reproduced locally with same settings. For demo project, give me a day or 2. I can create that and provide.
